onchaind/onchain.c

```diff
@@ -1980,8 +1980,8 @@ int main(int argc, char *argv[])
 				   &funding_amount_satoshi,
 				   &old_remote_per_commit_point,
 				   &remote_per_commit_point,
-				   &to_self_delay[LOCAL],
```

Comment:
Would it be better to add a comment here?

Future code divers might be confused why the below order is in the .csv file:

    # Remote per commit point for current tx (needed if we haven't got revoke_and_ack yet).
    onchain_init,,remote_per_commitment_point,struct pubkey
    onchain_init,,local_to_self_delay,u32
    onchain_init,,remote_to_self_delay,u32

and ordering in the code:

    				   &remote_per_commit_point,
    				   &to_self_delay[REMOTE],
    				   &to_self_delay[LOCAL],

e.g.: `/* to_self_delay[ _ ] is the configured to_self_delay setting, however we should apply the remote configuration to our local delay, and the local configuration to the remote delay. */`

Reply:
Agreed.  I've inverted the caller with a comment instead.